openc3/lib/openc3/io/json_drb_object.rb

```diff
@@ -58,20 +58,18 @@ def initialize(url: ENV['OPENC3_API_URL'], timeout: 1.0, authentication: nil)
     def method_missing(method_name, *method_params, **keyword_params)
       raise JsonDRbError, "Shutdown" if @shutdown
       @mutex.synchronize do
-        for attempt in 1..3
-          @log = [nil, nil, nil]
-          connect() if !@http
-          json_rpc_request = JsonRpcRequest.new(method_name, method_params, keyword_params, @id)
-          data = json_rpc_request.to_json(:allow_nan => true)
-          response_body = make_request(data: data)
-          if !response_body or response_body.to_s.length < 1
-            disconnect()
-          else
-            response = JsonRpcResponse.from_json(response_body)
-            return handle_response(response: response)
-          end
```

Comment:
No sleep delay in here ... why were we trying to retry here?

Reply:
The original COSMOS code had two attempts, in case the socket was broken by restarting the CmdTlmServer.   Not really relevant anymore and the sleep was really bad.  One of our junior engineers updated this.